src/main/resources/entity-types/invoice/drv_invoice_voucher_lines_details.json5

```diff
@@ -0,0 +1,209 @@
+{
+  id: '5db5fbd8-0dfa-4e87-a7fc-a3568e83effb',
+  name: 'drv_voucher_lines_details',
+  private: false,
+  fromClause: 'src_voucher_lines as vlines',
+  columns: [
+    {
+      name: 'id',
+      dataType: {
+        dataType: 'rangedUUIDType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      isIdColumn: true,
+      valueGetter: "vlines.id"
+    },
+    {
+      name: 'amount',
+      dataType: {
+        dataType: 'numberType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "vlines.jsonb ->> 'amount'"
+    },
+    {
+      name: 'createdDate',
```

Comment:
one more nit: for the fields in other entity type definitions, we've been using underscores instead of camel casing. We should probably do that here (and in the keys for the translations files) just to keep things consistent.